src/test/java/org/elasticsearch/count/simple/SimpleCountTests.java

```diff
@@ -177,4 +179,46 @@ public void localDependentDateTests() throws Exception {
             assertHitCount(countResponse, 20l);
         }
     }
+
+    @Test
+    public void testThatNonEpochDatesCanBeSearch() throws Exception {
```

Comment:
this test actually uncovered the above issue with the `QueryRangeParser` and numeric dates like `2015121212`, so I would prefer to leave it as it is